src/pages/HomePage/HomePage.js

```diff
@@ -43,7 +43,11 @@ export default function HomePage({
         onMouseUp={endTransition}
       >
         <SumCarbonFootPrint
-          sumCarbonFootprint={totalCarbonFootprint.toFixed(2) || 0}
+          sumCarbonFootprint={
+            (typeof totalCarbonFootprint === 'number' &&
```

Comment:
Kann man noch auslagern in eigene `getTotalCarbonFootprint()` und dort `typeof totalCarbonFootprint === 'number' && totalCarbonFootprint.toFixed(2)) || 0` kapseln.